Error reported by phpstan 0.12.92 which causes CI fail

```
$ make test-php-phpstan
composer install
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Warning: The lock file is not up to date with the latest changes in composer.json. You may be getting outdated dependencies. It is recommended that you run `composer update` or `composer update <package name>`.
Nothing to install, update or remove
Generating autoload files
phpstan composer.lock is not up to date.
composer bin phpstan install --no-progress
No composer.lock file present. Updating dependencies to latest instead of installing from lock file. See https://getcomposer.org/install for more information.
Loading composer repositories with package information
Updating dependencies
Lock file operations: 1 install, 0 updates, 0 removals
  - Locking phpstan/phpstan (0.12.92)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 1 install, 0 updates, 0 removals
  - Installing phpstan/phpstan (0.12.92): Extracting archive
Generating autoload files
1 package you are using is looking for funding.
Use the `composer fund` command to find out more!
php -d zend.enable_gc=0 vendor-bin/phpstan/vendor/bin/phpstan analyse --memory-limit=4G --configuration=./phpstan.neon --no-progress --level=5 appinfo lib
 ------ ------------------------------------------------------------------------------------------------------------ 
  Line   lib/Http/DownloadResponse.php                                                                               
 ------ ------------------------------------------------------------------------------------------------------------ 
  71     Parameter #2 $value of method OCP\AppFramework\Http\Response::addHeader() expects string, float|int given.  
 ------ ------------------------------------------------------------------------------------------------------------ 

                                                                                                                        
 [ERROR] Found 1 error                                                                                                  
                                                                                                                        

make: *** [Makefile:176: test-php-phpstan] Error 1
```

--- a/lib/Http/DownloadResponse.php
+++ b/lib/Http/DownloadResponse.php
@@ -19,7 +19,7 @@ use \OC\Files\View;
 class DownloadResponse extends \OCP\AppFramework\Http\Response {
 	private $request;
 	private $file;
-	
+
 	/**
 	 * @param IRequest $request
 	 * @param File $file
@@ -38,13 +38,13 @@ class DownloadResponse extends \OCP\AppFramework\Http\Response {
 			'content' => $this->file->getContent()
 		];
 		$size = \strlen($data['content']);
-		
+
 		if (isset($this->request->server['HTTP_RANGE']) && $this->request->server['HTTP_RANGE'] !== null) {
 			$isValidRange = \preg_match('/^bytes=\d*-\d*(,\d*-\d*)*$/', $this->request->server['HTTP_RANGE']);
 			if (!$isValidRange) {
 				return $this->sendRangeNotSatisfiable($size);
 			}
-			
+
 			$ranges = \explode(',', \substr($this->request->server['HTTP_RANGE'], 6));
 			foreach ($ranges as $range) {
 				$parts = \explode('-', $range);
@@ -68,7 +68,7 @@ class DownloadResponse extends \OCP\AppFramework\Http\Response {
 				$md5Sum = \md5($buffer);
 
 				// send the headers and data
-				$this->addHeader('Content-Length', $end - $start);
+				$this->addHeader('Content-Length', (string) ($end - $start));
 				$this->addHeader('Content-md5', $md5Sum);
 				$this->addHeader('Accept-Ranges', 'bytes');
 				$this->addHeader('Content-Range', 'bytes ' . $start . '-' . ($end) . '/' . $size);
@@ -78,14 +78,14 @@ class DownloadResponse extends \OCP\AppFramework\Http\Response {
 				return $buffer;
 			}
 		}
-		
+
 		$this->addHeader('Content-Type', $data['mimetype']);
 		$this->addContentDispositionHeader();
 		$this->addHeader('Content-Length', (string)$size);
 
 		return $data['content'];
 	}
-	
+
 	/**
 	 * Send 416 if we can't satisfy the requested ranges
 	 * @param integer $filesize
@@ -95,7 +95,7 @@ class DownloadResponse extends \OCP\AppFramework\Http\Response {
 		$this->addHeader('Content-Range', 'bytes */' . $filesize); // Required in 416.
 		return '';
 	}
-	
+
 	protected function addContentDispositionHeader() {
 		$encodedName = \rawurlencode($this->file->getName());
 		$isIE = \preg_match("/MSIE/", $this->request->server["HTTP_USER_AGENT"]);

